projects/novo-elements/src/styles/variables.scss

```diff
@@ -123,6 +123,7 @@ $entity-colors: (
   "earnCode": $color-earn-code,
   "billableCharge": $color-billable-charge,
   "payableCharge": $color-payable-charge,
+  "complianceManager": $color-positive,
```

Comment:
I think you need to follow the existing pattern of entity colors like invoiceStatement and include this in all three locations.

Reply:
This isnt technically an entity though. Also there are other things in that list that are only in that list and in no other lists. We only need it there to show the correct background color for a menu item